sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/models/HandleItem.java

```diff
@@ -266,9 +266,9 @@ public HandleItem setLastReconnectTime(OffsetDateTime lastReconnectTime) {
      *
      * @return the accessRightList value.
      */
-    public List<AccessRight> getAccessRightList() {
+    public List<ShareFileHandleAccessRights> getAccessRightList() {
```

Comment:
Method name should be updated to `getAccessRights` and param name should be `accessRights`. Also, update the javadocs.